src/ScheduleService.Data.Provider.MsSql.Ef/Migrations/20231217210755_InitialCreate.cs

```diff
@@ -70,7 +70,13 @@ private void CreateTableCategories(MigrationBuilder migrationBuilder)
       {
         Id = table.Column<Guid>(nullable: false),
         Name = table.Column<string>(nullable: false, maxLength: 20),
-        Color = table.Column<string>(nullable: false, maxLength: 50),
+        Color = table.Column<int>(nullable: false, maxLength: 50),
+        WorkspaceId = table.Column<Guid>(nullable: false),
+        CreatedBy = table.Column<Guid>(nullable: false),
+        CreatedAtUtc = table.Column<DateTime>(nullable: false),
+        ModifiedBy = table.Column<Guid>(nullable: true),
+        ModifiedAtUtc = table.Column<DateTime>(nullable: true),
+        IsActive = table.Column<bool>(nullable: false),
```

Comment:
It is better to replace it before CreatedBy, because 75-78 are service fields

Reply:
replaced